packages/react-native-reanimated/src/layoutReanimation/web/componentUtils.ts

```diff
@@ -148,10 +148,12 @@ export function setElementAnimation(
 ) {
   const { animationName, duration, delay, easing } = animationConfig;
 
-  element.style.animationName = animationName;
-  element.style.animationDuration = `${duration}s`;
-  element.style.animationDelay = `${delay}s`;
-  element.style.animationTimingFunction = easing;
+  requestAnimationFrame(() => {
```

Comment:
We definitely should add a comment here `requestAnimationFrame` is required here

Reply:
i agree! Done in [9a3ec76](https://github.com/software-mansion/react-native-reanimated/pull/6172/commits/9a3ec76202a25337992b5daa76ec95a096646464)